ambry-cloud/src/test/java/com.github.ambry.cloud/CloudBlobStoreTest.java

```diff
@@ -86,24 +93,41 @@ public void testStoreDeletes() throws Exception {
       addBlobToSet(messageWriteSet, size, Utils.Infinite_Time, refAccountId, refContainerId);
     }
     store.delete(messageWriteSet);
-    verify(dest, times(count)).deleteBlob(any(BlobId.class));
+    verify(dest, times(count)).deleteBlob(any(BlobId.class), eq(operationTime));
+  }
+
+  /** Test the CloudBlobStore updateTtl method. */
+  @Test
+  public void testStoreTtlUpdates() throws Exception {
+    MockMessageWriteSet messageWriteSet = new MockMessageWriteSet();
+    int count = 10;
+    for (int j = 0; j < count; j++) {
+      long size = 10;
+      long expirationTime = Math.abs(random.nextLong());
+      addBlobToSet(messageWriteSet, size, expirationTime, refAccountId, refContainerId);
+    }
+    store.updateTtl(messageWriteSet);
+    verify(dest, times(count)).updateBlobExpiration(any(BlobId.class), anyLong());
   }
 
   /** Test the CloudBlobStore findMissingKeys method. */
   @Test
   public void testFindMissingKeys() throws Exception {
```

Comment:
Could you add the case where deleteBlob() is called and the metadata is updated?